srcpkgs/The-Powder-Toy/template

```diff
@@ -24,8 +25,8 @@ fi
 do_install() {
 	vbin build/powder
 	vdoc README.md
-	vinstall resources/powder.desktop 644 /usr/share/applications/
-	for size in 16 24 32 48 128 256 ; do
-		vinstall resources/icon/powder-${size}.png 644 /usr/share/icons/hicolor/${size}x${size}/apps powder.png
+	vinstall build/resources/powder.desktop 644 /usr/share/applications/
+	for size in _16 _32 _48; do
+		vinstall resources/generated_icons/icon_exe${size}.png 644 /usr/share/icons/hicolor/${size}x${size}/apps powder.png
```

Comment:
if the sizes in the for loop are prefixed by `_`, the files won't be installed in a valid icon directory